ProcessMaker/Listeners/BpmnSubscriber.php

```diff
@@ -40,6 +40,40 @@
  */
 class BpmnSubscriber
 {
+    private $memory;
+
+    /**
+     * @param $element
+     * @param TokenInterface|null $token
+     * @return $this
+     */
+    public function registerErrorHandler($element, TokenInterface $token = null)
+    {
+        // This storage is freed on error (case of allowed memory exhausted)
+        $this->memory = str_repeat('*', 1024 * 1024);
+
+        $path = getcwd();
```

Comment:
Could you move the storage path, like: `storage_path("logs/unhandled_error.log")`